src/commonMain/kotlin/fr/acinq/lightning/channel/Commitments.kt

```diff
@@ -515,11 +508,11 @@ data class FullCommitment(
     val fundingTxId: ByteVector32 = commitInput.outPoint.txid
     val fundingAmount = commitInput.txOut.amount
     val localChannelReserve = when {
-        params.channelFeatures.hasFeature(Feature.ZeroReserveChannels) && !params.localParams.isInitiator -> 0.sat
+        params.channelFeatures.hasFeature(Feature.ZeroReserveChannels) -> 0.sat
```

Comment:
The duplication between `Commitments` and `FullCommitments` shows that there is some cleanup to do

Reply:
True, but I'm not sure how it could be cleaned up though, so I'm leaving it as-is for now.